apps/src/templates/projects/ProjectsList.jsx

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import {Table} from 'reactabular';
+import color from "../../util/color";
+
+const styles = {
+  cell: {
+    border: '1px solid gray',
+    padding: 10,
+    fontSize: 14,
+  },
+  nameCell: {
+    border: '1px solid gray',
+    padding: 10,
+    fontSize: 14,
+    fontFamily: '"Gotham 5r", sans-serif',
+  },
+  headerCell: {
+    border: '1px solid gray',
+    padding: 10,
+    backgroundColor: color.teal,
+  },
+};
+
+/**
+ * Map from project type to friendly name.
+ * @type {Object}
+ */
+const PROJECT_TYPE_MAP = {
+  applab: 'App Lab',
+  gamelab: 'Game Lab',
```

Comment:
I'm surprised this mapping doesn't already exist in an apps localization file.  Oddly enough to get here now I think you have to go [through game.rb](https://github.com/code-dot-org/code-dot-org/blob/956c585a077a3faa018e1af31d84da2c515d2ffa/dashboard/app/models/game.rb#L242) and [into data.en.yml](https://github.com/code-dot-org/code-dot-org/blob/3bcfcff5bdf4abff46523543b3104f36d56e1c77/dashboard/config/locales/data.en.yml#L49-L49).

Either way, feels like this should live somewhere more central?